x/slashing/keeper/hooks.go

```diff
@@ -64,6 +64,9 @@ func (h Hooks) AfterValidatorRemoved(ctx sdk.Context, consAddr sdk.ConsAddress,
 	h.k.AfterValidatorRemoved(ctx, consAddr)
 }
 
+func (h Hooks) BeforeTokenizeShareRecordRemoved(ctx sdk.Context, recordId uint64) {
```

Comment:
Can we leave a comment here that the empty function is just to fulfill the interface contract ?